src/nicolai_munsterhjelm_ex/ex01/comp_to_loop.py

```diff
@@ -0,0 +1,46 @@
+
+def squares_by_comp(n):
+    """
```

Comment:
Super with docstrings, though you do not always have to expand them, and a single line can suffice if it is relatively simple.